regressionTests/testjs/src/handlers/v1.0/transactionV1_0Handler.ts

```diff
@@ -1,6 +1,5 @@
-import {Err, Nonce, SimpleSigner} from "@elrondnetwork/erdjs/out";
+import {Nonce, SimpleSigner} from "@elrondnetwork/erdjs/out";
```

Comment:
Does it work with `"@elrondnetwork/erdjs";` (without `out`)?

Reply:
cannot be done right now because the `IProvider` interface is not exported

Reply:
OK. Will update this in a next version (after fixing erdjs).